.github/workflows/tests.yaml

```diff
@@ -537,10 +537,15 @@ jobs:
                       --target linux-x64-light-data-model-no-unique-id-${BUILD_VARIANT}-clang \
                       --target linux-x64-terms-and-conditions \
                       --target linux-x64-python-bindings \
+                      --target linux-x64-jf-control-app \
```

Comment:
Could you update https://github.com/project-chip/connectedhomeip/blob/master/scripts/tests/local.py as well so we can run all tests via this script?